client/game.js

```diff
@@ -64,10 +64,11 @@ const app = new Vue({
       );
     },
     canMoveTo(x, y) {
-      return (
-        distance(this.me.position, { x, y }) ==
-          this.config.Range.MoveDistance && this.me.health > 0
-      );
+      //checks if you can move forwards
+      straight = distance(this.me.position, { x, y }) == this.config.Range.MoveDistance
```

Comment:
Missing variable declaration here!
```suggestion
      const straight = distance(this.me.position, { x, y }) == this.config.Range.MoveDistance
```